exchanges/stats/stats.go

```diff
@@ -82,13 +40,14 @@ func Add(exchange string, p currency.Pair, a asset.Item, price, volume float64)
 	return nil
 }
 
-// Append adds or updates the item stats for a specific
+// Append adds or updates the Item stats for a specific
 // currency pair and asset type
 func Append(exchange string, p currency.Pair, a asset.Item, price, volume float64) {
```

Comment:
can unexport this

Reply:
The problem with unexporting functions like this is that I can't know what external systems use